### DISCUSSION TITLE
Sync with third-party API/Database?

### DISCUSSION THREAD

**Discussion by oguruma1218** [supabase/supabase#18625] [Questions]
I'd like to find a way to periodically (every few hours, maybe) sync a Supabase table with a third-party API. The reason why it's not feasible to use the third-party API, directly, is that it is far too slow, with queries taking in excess of 10 seconds most of the time. Is there a way within Supabase to have something akin to a cron job run periodically to populate the database periodically?

**Comment by GaryAustin1** [comment]
With these two extensions you can do it.
https://supabase.com/docs/guides/database/extensions/pg_cron
https://supabase.com/docs/guides/database/extensions/http
